tests/chaos/conftest.py

```diff
@@ -188,23 +188,21 @@ def rebooting_control_plane_node(
 def pod_deleting_process(request, admin_client):
     pod_prefix = request.param["pod_prefix"]
     namespace_name = request.param["namespace_name"]
-    pod_deleting_process = create_pod_deleting_process(
+    process = create_pod_deleting_process(
         dyn_client=admin_client,
         pod_prefix=pod_prefix,
         namespace_name=namespace_name,
         ratio=request.param["ratio"],
         interval=request.param["interval"],
         max_duration=request.param["max_duration"],
     )
-    pod_deleting_process.start()
-    yield pod_deleting_process
-    terminate_process(process=pod_deleting_process)
-
-    pod_deleting_process_recover(
-        resource=request.param["resource"],
-        namespace=namespace_name,
-        pod_prefix=pod_prefix,
-    )
+    process.start()
+    yield {
+        "process": process,
```

Comment:
you don't need process. It is not used anywhere.

Reply:
Done